scipy/ndimage/src/_ni_measure.pyx

```diff
@@ -0,0 +1,173 @@
+######################################################################
+# Cython version of scipy.ndimage.measurement.find_objects() function.
+######################################################################
+
+cimport cython
+from cython cimport sizeof
+import numpy as np
+cimport numpy as np
+
+np.import_array()
+
+cdef extern from *:
+   ctypedef int Py_intptr_t
+
+ctypedef void (*PyArray_CopySwapFunc)(void *, void *, int, void *)
+
+cdef extern from "numpy/arrayobject.h" nogil:
+    ctypedef struct PyArrayIterObject:
+        np.intp_t *coordinates
+        char *dataptr
+        np.npy_bool contiguous
+
+    ctypedef struct PyArrayObject:
+        int nd
+
+    void *PyDataMem_NEW(size_t)
+    void PyDataMem_FREE(void *)
+
+
+######################################################################
+# Use of Cython's type templates for type specialization
+######################################################################
+
+# Only integer values are allowed.
+ctypedef fused data_t:
+    np.int8_t
+    np.int16_t
+    np.int32_t
+    np.int64_t
+    np.uint8_t
+    np.uint16_t
+    np.uint32_t
+    np.uint64_t
+
+#####################################################################
+# Function Specializers and asociate function for using fused type
+#####################################################################
+
+ctypedef void (*func_p)(void *data, np.flatiter iti, np.intp_t max_label, 
+               int *regions, int rank) nogil
+
+def get_funcs(np.ndarray[data_t] input):
+    return (<Py_intptr_t> findObjectsPoint[data_t])
```

Comment:
@jaimefrio That is required. Removing it from line 53 gives error while compiling the code and, in line 103 removing it gives segmentation fault.